pypsa/plot.py

```diff
@@ -507,6 +507,10 @@ def compute_bbox_with_margins(margin, x, y):
     """
     Helper function to compute bounding box for the plot.
     """
+    # Drop NAs to avoid issues in calculation of bbox
+    x.dropna(inplace=True)
+    y.dropna(inplace=True)
+
     # set margins
     pos = np.asarray((x, y))
     minxy, maxxy = pos.min(axis=1), pos.max(axis=1)
```

Comment:
```suggestion
    minxy, maxxy = pos.nanmin(axis=1), pos.nanmax(axis=1)
```

This should also work?

Reply:
Good proposal, however, np.nanmin() and np.nanmax() need to be used then --> implemented:
```
def compute_bbox_with_margins(margin, x, y):
    """
    Helper function to compute bounding box for the plot.
    """
    # set margins
    pos = np.asarray((x, y))
    minxy, maxxy = np.nanmin(pos, axis=1), np.nanmax(pos, axis=1)
    xy1 = minxy - margin * (maxxy - minxy)
    xy2 = maxxy + margin * (maxxy - minxy)
    return tuple(xy1), tuple(xy2)
```